Makefile

```diff
@@ -580,13 +624,20 @@ release-once-tagged:
 	@echo "Will now build release artifacts..."
 	@echo
 	$(MAKE) bin/calico-felix calico/felix
-	docker tag calico/felix:latest quay.io/calico/felix:latest
+	# default image until we use multi-arch manifest
```

Comment:
> Does running make release-once-tagged ARCH=ppc64le work, for example?

It does. More correctly, it runs all of the correct targets with the correct vars.

Reply:
`make release` should be fine. All it does is change the release notes and add them as a tag, which has no impact on the compiled binaries or images.

Reply:
> Given how many images there now are, it might be good to have a make push VERSION=3.0.0 target to do all the pushing

Yes. I didn't know why it didn't have one. I assumed there was some reason to which I wasn't to privy. Separate PR though. The rebasing has been so much work with each change, and this one open for so long, we should get it in, then do a simple PR for that.

Reply:
Agreed